samples/amcache.py

```diff
@@ -15,10 +15,13 @@
 #   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
+
+from __future__ import print_function
 import sys
 import logging
 import datetime
 from collections import namedtuple
+import json
```

Comment:
style nit pick: i like to order my imports by line length (with group of stdlib, pip-installable, and project-local modules).